### DISCUSSION TITLE
Comparing int value states

### DISCUSSION THREAD

**Discussion by kristianolofsson** [Mariusthvdb/custom-ui#77] [General]
Hi,

I installed the HACS Custom-ui and it works perfectly when using a string state. Like below:
<img width="356" alt="Screenshot 2022-12-16 at 19 41 02" src="https://user-images.githubusercontent.com/7191232/208167062-9368bb3f-af99-4f4c-b9d9-fcb24f2e6545.png">

But when trying the same for the light counter sensor i cant get it to work.
<img width="800" alt="Screenshot 2022-12-16 at 18 09 37" src="https://user-images.githubusercontent.com/7191232/208167328-c6fe4aab-2754-4a51-a5d8-b2c205afd7d4.png">

I tried the with and without quotes
<img width="318" alt="Screenshot 2022-12-16 at 19 41 21" src="https://user-images.githubusercontent.com/7191232/208167555-bab3cb5a-e992-4759-8463-d8479c34c2c5.png">

**Comment by Mariusthvdb** [comment]
the bottom image: is that the actual yaml in  file? because that cant be correct.
the top is supposed to be under

```
homeassistant:
  customize:
```

and, also the bottom is indented incorrectly, as you can see on the dotted lines

**Reply by kristianolofsson** [reply]
Correct the bottom customize file carries the correct top lines. All other sensors works.

and what’s wrong with the template? It counts the lights on in the light.group. Should I combine the template with color of icon?

**Reply by Mariusthvdb** [reply]
Please don't post pictures of yaml...

There is no

``` templates: ``` 

Right above the bottom ``` icon_color: ```
